playground/frontend/playground_components/lib/src/enums/complexity.dart

```diff
@@ -20,5 +20,19 @@ enum Complexity {
   unspecified,
   basic,
   medium,
-  advanced,
+  advanced;
```

Comment:
It's better to format like this so we can easily add new constants after `advanced`:
```dart
  advanced,
  ;
```